### PR TITLE
Updates CI configuration

### PR DESCRIPTION
**Pull request by HaSistrunk**
- Updates README to follow our template.
- Simplifies Travis CI configs.
- Adjusts htmlproofer configs to check external links in cibuild.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,7 +18,5 @@ addons:
     packages:
     - libcurl4-openssl-dev
 
-cache: bundler # caching bundler gem packages will speed up build
```
Comment (helrond) [inline .travis.yml:23]: We *do* want to cache!

**Review by helrond — CHANGES_REQUESTED** [review]
Not sure if there's a reason caching is disabled on Travis configs, but I think we want to enable that to help speed things up.

File: .travis.yml (hunk 1 above)
Comment (HaSistrunk) [reply .travis.yml:23]: RIGHT. But also, did you disable caching in the ci branch PRs for rockarch.org and Bits & Bytes?

Comment (HaSistrunk) [reply .travis.yml:23]: RIGHT

**Review by helrond — APPROVED** [review]

**Review by p-galligan — APPROVED** [review]

**Review by bonniegee — APPROVED** [review]

**Pull request merged**